### ISSUE TITLE
I need help

### ISSUE THREAD

**Issue by DRCRecoveryData** [spectran/CDJ-100S-STM32F7-Display#2]
Hi,

Is that STMF103 use as midi controller and connect to STM Disco Board via SPI?

I have a ddj-400 and wanna made a screen mod on it

**Comment by spectran** [comment]
Yes, STM32F103 sends both MIDI commands to USB (if the USB interface is connected), and SPI commands to STM32F746G-Discovery board.
You decided to attach display to DDJ-400? I think, you can use STM32F746G-Discovery, but you need to modify code, because it have to receive MIDI commands from DDJ-400. Then, DDJ-400 will send MIDI commands to STM32F746G-Disco, and the board will play songs from SD card inserted.
